samples/calculator/evals/evaluators/correct-operator-evaluator.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "1.0",
+  "id": "CorrectOperatorEvaluator",
+  "evaluatorTypeId": "file://types/correct-operator-evaluator-types.json",
+  "evaluatorSchema": "file://correct_operator.py:CorrectOperatorEvaluator",
```

Comment:
`file://` typically starts with root. Regardless, should we point this to an import module like `module.submodule:ClassName`?